it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java

```diff
@@ -87,6 +93,46 @@ public void test() throws Exception {
 
         assertThat(property.getBar()).isEqualTo("Y");
         assertThat(property.getQux()).containsExactly("0", "1");
+        assertThat(property.getRevision()).isNotNull();
+        assertThat(called[0]).isEqualTo(1);
+
+        // test that after close a watcher, it could not receive change anymore
+        property.closeWatcher();
+        client.push("a", "b", Revision.HEAD, Author.SYSTEM, "Modify a.json",
+                    Change.ofJsonUpsert("/c.json",
+                                        '{' +
+                                        "  \"foo\": 50," +
+                                        "  \"bar\": \"Y2\"," +
+                                        "  \"qux\": [\"M\", \"N\"]" +
+                                        '}'))
+              .join();
+        // TODO(huydx): this test may be flaky, is there any better way?
+        Throwable thrown = catchThrowable(() -> await().atMost(2, TimeUnit.SECONDS)
```

Comment:
I think it's fine. Perhaps we don't even need to specify the timeout (2 seconds)? The default timeout of 10 seconds should be OK.

Reply:
Let me merge once you remove `.atMost()`. Great job!

Reply:
this test will always await, so if we use default 10 seconds, each developer will need to wait 10sec everytime, is that ok?